pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1904,6 +2037,48 @@ func (m *kubeGenericRuntimeManager) IsPodResizeInProgress(allocatedPod *v1.Pod,
 		})
 }
 
+func (m *kubeGenericRuntimeManager) isPodLevelResourcesResizeInProgress(allocatedPod *v1.Pod, podStatus *kubecontainer.PodStatus) bool {
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodLevelResourcesVerticalScaling) {
+		return false
+	}
+
+	if allocatedPod.Spec.Resources == nil {
+		return false
+	}
+
+	actuatedPodResources, _ := m.actuatedState.GetPodLevelResources(allocatedPod.UID)
+	allocatedPodResources := allocatedPod.Spec.Resources
+
+	return cpuMemoryResourcesEqual(actuatedPodResources, allocatedPodResources)
```

Comment:
This is inverted. 
```suggestion
	return !cpuMemoryResourcesEqual(actuatedPodResources, allocatedPodResources)
```

Here's unit test coverage: https://github.com/tallclair/kubernetes/commit/e21d01fdba159b414e3fcf40f8692355c9569e1b

Reply:
looking into it